Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1739,6 +1745,12 @@ static String mutationObserverNotificationString()
     return m_frame->isMainFrame() && isFullFramePlugin();
 }
 
+bool UnifiedPDFPlugin::shouldCachePagePreviews() const
+{
+    // Only main frame plugins are hooked up to releaseMemory().
+    return isFullFramePlugin();
```

Comment:
I believe we want `isFullMainFramePlugin` here, right?

Reply:
No, `isFullFramePlugin()` is correct, because the `releaseMemory` code talks to all Documents.